useful-links/useful-links.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="useful-links.css" />
     <title>Useful Links</title>
   </head>
+
```

Comment:
we do not use white space between lines of code

Reply:
Ohh, my bad, I've use to seeing white spaces :)